STEPS.md

```
@@ -828,6 +829,43 @@ Then the downloaded file should be a zip archive not containing the files partia
 
 </details>
 
+## JavascriptTrait
+
+[Source](src/JavascriptTrait.php), [Example](tests/behat/features/javascript.feature)
+
+>  Automatically detect JavaScript errors during test execution.
```

markdownlint-cli2 (0.18.1)

Multiple spaces after blockquote symbol

(MD027, no-multiple-space-blockquote)